pom.xml

```diff
@@ -175,6 +175,16 @@
 			<artifactId>hibernate-c3p0</artifactId>
 			<version>${hibernate.version}</version>
 		</dependency>
+		<dependency>
+			<groupId>org.hibernate.search</groupId>
+			<artifactId>hibernate-search-mapper-orm</artifactId>
+			<version>6.1.8.Final</version>
+		</dependency>
+		<dependency>
+			<groupId>org.hibernate.search</groupId>
+			<artifactId>hibernate-search-backend-lucene</artifactId>
+			<version>6.1.8.Final</version>
+		</dependency>
```

Comment:
Thanks @rahul6603 .
Since these are related ,you can define a [property variable](https://github.com/I-TECH-UW/OpenELIS-Global-2/blob/develop/pom.xml#L12-L29) for this version and just resue it here